peps/pep-0772.rst

```diff
@@ -184,201 +184,196 @@ making process that governs the technical projects under the PyPA umbrella.
+defined in the `PSF Bylaws <https://www.python.org/psf/bylaws/>`_.  Should those bylaws change in the future,
+the eligibility of Packaging Council Electors will similar change to match.  As with PSF voting membership,
+Packaging Council Electors must affirm their intention to vote in Packaging Council elections every year.
 
 .. _process:
 
 Processes
 =========
 
-Initial membership
-------------------
-
-Initial set of voting members will include anyone who has taken the time to
-formalise their participation in the packaging community. This includes:
-
-* PyPA members: Anyone with the triage bit or commit bit, or at least one
-  project in the PyPA organisation.
-* Packaging workgroup members: Anyone who is listed on the Packaging WG charter
-  who is willing to participate is welcome.
-* Interested core team members: Any Python core team member who is willing to
-  participate is welcome.
-* Wider community members: An initial set of for-profit companies, nonprofit
-  organizations, academic or educational institutions and smaller unaffiliated
-  projects would be invited to nominate three individuals to represent them.
-  See :ref:`the initial set of community voting members <appendix_C>`.
-
-Adding a new member
--------------------
-
-New voting members are added by a simple majority vote by the current
-membership. Quorum for adding new members is 50%.
-
-A vote to add a new member is triggered when a voting member calls for one
-publicly on an appropriate communication channel, and another voting member
-seconds the call within two weeks.
-
-The vote lasts for one week. Each member can vote for or against.
-
 Removal of a member
 -------------------
 
-In order to maintain a reasonable expectation of quorum, failure to participate
-in Packaging Council elections for two consecutive council elections
-automatically removes a person from the list of voting members, until they
-re-submit their intention to resume their participation to the Packaging
-Council in writing.
+In order to maintain a reasonable expectation of quorum, failure to participate in Packaging Council elections
+for two consecutive Council elections automatically removes a person from the list of Packaging Council
+Electors, until they re-submit their intention to resume their participation to the Packaging Council in
+writing.
 
-In exceptional circumstances, it may be necessary to remove someone from the
-voting members against their will (for example: egregious and ongoing code of
-conduct violations). A voting member may be removed by a two-thirds majority
-vote by the Packaging Council (in practice: 4:1 for a council with five
-members).
+In exceptional circumstances, it may be necessary to remove someone from the Electors against their will (for
+example: egregious and ongoing code of conduct violations). An Elector may be removed by a two-thirds majority
+vote by the Packaging Council (in practice: 4:1 for a Council with five members).
```

Comment:
```suggestion
In exceptional circumstances, it may be necessary to remove someone from the Electors against their will (for
example: egregious and ongoing code of conduct violations). An Elector may be removed by a two-thirds majority
vote by the Packaging Council (in practice: 4:1 for a Council with five members).

In such a case, the former Elector may not rejoin unless approved by another
two-thirds majority vote by the Packaging Council.
```

Reply:
It occurs to me this comes from the PEP 13 text about removing a core dev, which is important because being a core dev is a role with responsibilities in its own right, not just electing the SC. Since the only thing an Elector can do is vote people onto or off the Packaging Council, since the goal is to have a lot of Electors so no single vote is particularly significant, and since there is already a mechanism for removal of PSF membership ([bylaws](https://www.python.org/psf/bylaws/) 4.15, 2/3 of PSF voting members present at a meeting or majority of the Board), what if we just strike this whole section?

Reply:
WFM!